gulpfile.babel.js

```diff
@@ -267,7 +267,6 @@ gulp.task('metadata', () => {
 gulp.task('default', ['clean'], cb => {
   runSequence(
     ['styles', 'styles-grid'],
-    ['scripts'],
```

Comment:
Then I wonder how we ended up getting styles generated 4 times before if gulp will de-dup dependencies. I'll set it up with `scripts` still in the chain and test manually to verify this behavior, because from what we have seen before I don't trust it with our process.

I'll check up on this in the morning. Thanks for the minimal example to show it does function like that though.